pom.xml

```diff
@@ -10,7 +10,7 @@
         <jdk.version>1.6</jdk.version>
         <main.class>org.blondin.fa.Main</main.class>
     </properties>
-    <build>
+    <build>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this line start at column 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarsource.org/coding_rules#rule_key=xml%3AIndentCheck)